picasso/src/main/java/com/squareup/picasso3/Request.kt

```diff
@@ -330,6 +337,7 @@ class Request internal constructor(builder: Builder) {
       priority = request.priority
       memoryPolicy = request.memoryPolicy
       networkPolicy = request.networkPolicy
+      headers = request.headers
```

Comment:
Headers were also not being copied when we utilize newBuilder() like in BitmapHunter

```
    if (retryCount == 0) {
      data = data.newBuilder().networkPolicy(NetworkPolicy.OFFLINE).build()
    }
```

Reply:
good catch!